packages/support/resources/views/components/dropdown/index.blade.php

```diff
@@ -21,7 +21,7 @@
     }"
 >
     <div
-        class="filament-dropdown-trigger"
+        class="filament-dropdown-trigger cursor-pointer"
```

Comment:
@danharrin I left this out on purpose: https://github.com/tailwindlabs/tailwindcss/pull/8962.

Reply:
The rest of our buttons have that behaviour so I added it for consistency.

Reply:
Understand. Let's discuss this for v3?